docs/en/deployment.md

```diff
@@ -9,199 +9,151 @@ Before you launch the solution, review the architecture, supported regions, and
 
-* Sign in to the AWS Management Console, navigate to [QuickSight](https://quicksight.aws.amazon.com/).
 
-* If you did not have a QuickSight account, you need to sign up for QuickSight.
+## Step 1: Prerequisites
 
-* Choose **Enterprise**, click continue.
+### Enable Amazon Braket service
 
-* In the **Create your QuickSight account** page, fill the necessary information:
+1. Login AWS Console, find Amazon Braket.
 
-<center>
+2. Open Amazon Braket, select **I have read and accepted the above terms & conditions**.
 
-|      Field Name      |   Value |  Comment |
-|:--------------------:|:-------------------:| :-------------------:|
-| Authentication method | Choose **Use IAM federated identities & QuickSight-managed users** | The default value|
-| Select a region | Choose **US East(N.Virginia)**  | The default value |
-| QuickSight account name | Any string value | Unique QuickSight account name|
-| Notification email address | your email | |
-|IAM Role| Choose **Use QuickSight-managed role (default)**| The default value |
-|Allow access and autodiscovery for these resources| Choose  **IAM**, **Amazon Athena**| The default value |
+3. Click **Enable Amazon Braket**.
 
-</center>
+### Create IAM Role for QuickSight
 
+1. Go to [IAM Policies](https://console.aws.amazon.com/iamv2/home?#/policies).
 
-### Update QuickSight permissions
+2. Click **Create Policy**.
 
-* Go to [IAM Roles](https://console.aws.amazon.com/iamv2/home?#/roles).
+3. In Create policy page, click **JSON** tab, fill [QuickSight policy](quicksight-policy.json).
 
-* Search role by prefix: `aws-quicksight-service-role`, you should find one result, e.g. `aws-quicksight-service-role-v0`
+4. Click **Next:Tags**.
 
-* Click the role name.
+5. Click **Next:Review**.
 
-* Click **Add permissions**, in the drop down list, click **Create inline policy**.
+6. Fill **Name**: `qradd-quicksight-service-role-policy`.
 
-* Click **JSON** tab, pasted below json into it.
+7. Click **Create policy**.
 
-```json
-{
-    "Version": "2012-10-17",
-    "Statement": [
-         {
-            "Effect": "Allow",
-            "Action": "s3:ListAllMyBuckets",
-            "Resource": "arn:aws:s3:::*"
-        },
+8. Go to [IAM Roles](https://console.aws.amazon.com/iamv2/home?#/roles).
 
-        {
-            "Action": [
-                "s3:ListBucket"
-            ],
-            "Effect": "Allow",
-            "Resource": [
-                "arn:aws:s3:::amazon-braket-qcstack-*"
-            ]
-        },
+9. Click **Create Role**.
 
-         {
-            "Action": [
-                "s3:GetObject",
-                "s3:GetObjectVersion"
-            ],
-            "Effect": "Allow",
-            "Resource": [
-                "arn:aws:s3:::amazon-braket-qcstack-*/*"
-            ]
-        }
-    ]
-}
+10. Select **Custom trust policy** in the Select trusted entity page, fill [QuickSight trust policy](quicksight-trust-policy.json) in **Custom trust policy**.
```

Comment:
I prefer to use inline code snippet instead of a file.

Reply:
changed.